Makefile

```
@@ -0,0 +1,21 @@
+
+NAME=terraform-provider-unifi
+PLUGIN_PATH=$(HOME)/.terraform.d/plugins
+
+all: build
+
+build:
+	go build -o $(NAME)
+
+
+install: build
+	install -d $(PLUGIN_PATH)
+	install -m 775 $(NAME) $(PLUGIN_PATH)/
+
+test: install
```

I'm good with a Makefile and the rest of this, but I don't typically install it in this way and wouldn't want to couple testing with installing as I don't think its necessary right?
```suggestion
test:
```